packages/suite/src/views/wallet/send/components/Outputs/components/Amount/index.tsx

```diff
@@ -242,21 +254,23 @@ const Amount = ({ output, outputId }: Props) => {
                 <StyledInput
                     inputState={getInputState(error, amountValue)}
                     isMonospace
-                    labelAddonIsVisible={isSetMaxActive}
+                    labelAddonIsVisible={isSetMaxVisible}
                     labelAddon={
-                        isSetMaxVisible ? (
-                            <Button
-                                icon={isSetMaxActive ? 'CHECK' : 'SEND'}
-                                data-test={`outputs[${outputId}].setMax`}
-                                onClick={() => {
+                        <SwitchWrapper>
+                            <Switch
```

Comment:
Animation is not working for me here, don't know why. I can see the animation on other Switches i.e labelling switch in Settings.

Reply:
Looks like it was rendering another element instead of just changing style, so it skipped the transition. Fixed in 4269217cb99747ec88b6919d4e73feaff6bda0fe.

Reply:
nice 😉